site/app/templates/officeHoursQueue/CurrentQueue.twig

```diff
@@ -23,19 +23,19 @@
                 <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
                 <input type="hidden" name="user_id" value="{{entry['user_id']}}"/>
                 {% if entry['paused'] %}
-                  <button type="submit" class="btn btn-primary help_btn" title="Student paused">
+                  <button type="submit" class="btn btn-primary help_btn" title="Student paused" ready="false">
```

Comment:
We should use `data-ready="false"` and `data-ready="true"` here. Generally, we want to avoid introducing non-standard attributes for a HTML tag, preferring to use the [data-* attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) to define custom data on elements.